rails/init.rb

```diff
@@ -1,16 +1,16 @@
 require "pagem"
 
-['/public/stylesheets/pagem.css'].each do |file|
-    source = File.join(directory,file)
-    dest = RAILS_ROOT + file
-    FileUtils.cp(source, dest)
-end
-
-
-# move images to public/images/medidata_buttons
-['/public/images/pagem'].each do |dir|
-  source = File.join(directory,dir)
-  dest = RAILS_ROOT + dir
-  FileUtils.mkdir_p(dest)
-  FileUtils.cp(Dir.glob(source+'/*.*'), dest)
-end
\ No newline at end of file
+# ['/vendor/assets/stylesheets/pagem.css'].each do |file|
+#     source = File.join(directory,file)
+#     dest = RAILS_ROOT + file
+#     FileUtils.cp(source, dest)
+# end
+#
+#
+# # move images to vendor/assets/images/medidata_buttons
+# ['/vendor/assets/images/pagem'].each do |dir|
+#   source = File.join(directory,dir)
+#   dest = RAILS_ROOT + dir
+#   FileUtils.mkdir_p(dest)
+#   FileUtils.cp(Dir.glob(source+'/*.*'), dest)
+# end
```

Comment:
Asset pipeline by default looks on ```vendor/assets``` path for images, stylesheets, etc., from within the gem itself.  There is not need to copy the files into the project itself.